Sources/Build/BuildPlan.swift

```diff
@@ -1200,6 +1200,21 @@ public final class ProductBuildDescription {
         return args.filter({ !invalidArguments.contains($0) })
     }
 
+    private var deadStripArguments: [String] {
+        switch buildParameters.configuration {
+        case .debug:
+            return []
+        case .release:
+            if buildParameters.triple.isDarwin() {
+                return ["-Xlinker", "-dead_strip"]
+            } else if !buildParameters.triple.isWindows() {
+                return ["-Xlinker", "--gc-sections"]
+            } else {
```

Comment:
Please add a case for windows, the flag should be `/OPT:REF`.

Reply:
Thanks, lmk if this update looks right!